tokio/src/io/util/read_until.rs

```diff
@@ -8,27 +8,30 @@ use std::task::{Context, Poll};
 
 cfg_io_util! {
     /// Future for the [`read_until`](crate::io::AsyncBufReadExt::read_until) method.
+    /// The delimeter is included in the resulting vector.
```

Comment:
This struct is not exported, so could you move this description to the `read_until` method?

Reply:
That method already has this in the documentation:

> This function will read bytes from the underlying stream until the delimiter or EOF is found. Once found, all bytes up to, and including, the delimiter (if found) will be appended to buf.

Reply:
Oh, you are right, I missed it for some reason.